R/forecast2.R

```diff
@@ -80,6 +80,7 @@ meanf <- function(y,h=10,level=c(80,95),fan=FALSE, lambda=NULL, biasadj=FALSE,
     e <- na.omit(res) - mean(res, na.rm=TRUE)
     sim <- matrix(sample(e, size=npaths*h, replace=TRUE), ncol=npaths, nrow=h)
     sim <- sweep(sim, 1, f, "+")
+    s <- mean(apply(sim, 2, sd))
```

Comment:
s should be the sd of the historical data. Just need to move the line defining s to before if(bootstrap).